tests/topotests/lib/pim.py

```diff
@@ -1171,16 +1175,18 @@ def verify_upstream_iif(
                         if group_addr_json[src_address]["regState"] != regState:
                             errormsg = (
                                 "[DUT %s]: Verifying iif "
-                                "(Inbound Interface) for (%s,%s) and"
-                                " rejstate :%s [FAILED]!! "
-                                " Expected: %s, Found: %s"
+                                "(Inbound Interface) and rejstate "
```

Comment:
Should it be rejstate or regstate in real? :smile:

Reply:
It should be regState, Thank you